### ISSUE TITLE
Should genera of lattices have home pages?

### ISSUE THREAD

**Issue by kedlaya** [LMFDB/lmfdb#748]
Right now, there is a search page for all lattices and individual home pages for individual lattices. Should genera of lattices also have home pages?

By way of comparison, elliptic curves over Q come in isogeny classes, and both the individual curves and the isogeny classes have home pages (which are friends of each other).

The analogy is not perfect, because for elliptic curves, any two curves in the same isogeny class have the same L-function, whereas lattices in the same genera do not. On the other hand, the genus itself does have an L-function associated to it, namely the sum of the individual L-functions of the lattices in the genus, and that is actually better behaved than the individual ones (it has an Euler product).

Anyway, this issue should be closed either if it is decided not to pursue this, or if a first working version of genus home pages is created (at which point issues with them should be raised separately).

**Comment by davidfarmer** [comment]
Certainly this should be done, but not necessarily as a priority.
I have added appropriate labels.

**Comment by sanni85** [comment]
I like this idea. At the moment if you click on the 1 after class number on the lattice homepage http://beta.lmfdb.org/Lattice/ you get the list of all lattices of class number 1, which are the only one in the database at the moment. I will try to work on this in the next days and comment on it here.

**Comment by annahaensch** [comment]
I think perhaps a more interesting thing would be to add lattices in the same genus (these would be the ones given by "genus representatives") as friends of any given lattice. Or perhaps design it so you can click on any of the genus representatives to arrive at that lattice's homepage.

**Comment by jvoight** [comment]
Wait, are lattices not organized in their labels according to genus?  Is the "class number" the sum of the class numbers over the genus?  So if I have several genera with the same dimension, determinant, level, they're all lumped together?

If I'm not mistaken, as above this would be like taking all elliptic curves of conductor N and putting them all in the same box as opposed to labeling/organizing them according to isogeny class.  This seems very suboptimal.

**Comment by JohnCremona** [comment]
I have often wondered if we want a new page for elliptic curves showing all those of a fixed conductor, displaying minimal data and split into isogeny classes.  To replace (e.g.) http://www.lmfdb.org/EllipticCurve/Q/37/  which is just a special search results table.

Just a remark.

**Comment by annahaensch** [comment]
You are correct @jvoight the labeling is really suboptimal right now, and yes the forms of same determinant, level, and class number are lumped together.  This is precisely my interest in the canonical labeling from Matthieu, so that we can use genus symbols to group by genus, and then label within genus in some way that makes sense.  I think that @sehlen (Stephan Ehlen) has the genus symbol part ready to go, but we still need to implement the intra-genus labeling.  But yes, that should get some attention. @sanni85

**Comment by sehlen** [comment]
At our last lmfdb lattices meeting we discussed the labels quite a bit. And indeed, one of the most important changes we all agreed on was that the genus should be part of the label and this can be done canonically via genus symbols (some problems for the prime 2 occur but that can be solved with a convention). However, we didn't implement this yet because we wanted the last bit of the label, the counter for each lattice within a class to be canonical (and by that I really mean completely reproducible). Maybe we should have updated the labels already at this stage but changing labels is something we want to avoid so I guess we only wanted to do it once but then right. What is the status, @annahaensch ? Is there in fact a paper by Matthieu that we could look at? If you have any input for us, @jvoight, that would of course also be highly appreciated.

**Comment by tdupu** [comment]
Has there been any updates on this?
